canonical/canonical.com
# feat: /solutions/open-ran build

## Done

- Built new page as per [doc](https://docs.google.com/document/d/1rqBh8oVNeDk-fg1jg-7-SErLgmi0naEzP4r50JhJUb4/edit?tab=t.0) and [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=703-7693&node-type=instance&m=dev)
- Drive by:
  - Updated spacing on /solutions/telco hero per UX request  

## QA

- View the site locally in your web browser at: https://canonical-com-1400.demos.haus/solutions/telco/open-ran
- Check against doc and design
- https://canonical-com-1400.demos.haus/solutions/telco and see that the hero spacing has been updated

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14843
